app/src/main/java/de/danoeh/antennapod/adapter/ChaptersListAdapter.java

```diff
@@ -109,7 +109,7 @@ public View getView(final int position, View convertView, ViewGroup parent) {
                     if (link.length != 0) {
```

Comment:
Not sure about this change. For one, I think the old version is actually clearer - you would have to know that there are (currently!) only two pressure gestures.
This could also change in the future when Google copies that 3D touch peak stuff.